src/DELETE_THIS_FILE

```diff
@@ -0,0 +1 @@
+
```

Comment:
Delete this file

Reply:
I actually don't have the permission to delete.

Reply:
Delete it locally and commit the code it will work.